src/scripts/ui/GrandBazaarBuildingBody.tsx

```diff
@@ -163,9 +177,16 @@ export function GrandBazaarBuildingBody({ gameState, xy }: IBuildingComponentPro
             compareFunc={(a, b, i) => {
                switch (i) {
                   case 0:
-                     return (a.sellAmount ?? 0) - (b.sellAmount ?? 0);
+                     return Config.Resource[a.sellResource]
+                        .name()
+                        .localeCompare(Config.Resource[b.sellResource].name());
                   case 1:
-                     return (a.buyAmount ?? 0) - (b.buyAmount ?? 0);
+                     if (buyResourceFilter != null) {
```

Comment:
This changes the behavior of sorting based on whether a filter is applied or not. I am not sure it is a good design. I think consistency is more important here.

Reply:
I'll just change it to names.